pkg/controllers/vscr/vdbverify_reconciler.go

```diff
@@ -108,6 +108,15 @@ func (s *VDBVerifyReconciler) checkVersionAndDeploymentType(ctx context.Context)
 			events.VclusterOpsScrutinizeNotSupported, "NotReady:IncompatibleDB")
 	}
 
+	if vinf.IsOlder(v1.ScrutinizeLogAgeVersion) {
+		ver, _ := s.Vdb.GetVerticaVersionStr()
+		s.VRec.Eventf(s.Vscr, corev1.EventTypeWarning, events.VclusterOpsScrutinizeNotSupported,
+			"The server version %s does not support scrutinize with a log time range. The minimum server version it supports is %s.",
+			ver, v1.ScrutinizeLogAgeVersion)
+		return s.updateStateAndScrutinizeReadyCondition(ctx, metav1.ConditionFalse,
+			events.VclusterOpsScrutinizeNotSupported, "NotReady:IncompatibleDB")
+	}
+
```

Comment:
This block makes sense only when at least one of the logAge annotations is set. Otherwise we can skip this check.
Don't you think?